src/elf/reloc.rs

```diff
@@ -1,48 +1,70 @@
 //! # Relocation computations
+//!
+//! The following notation is used to describe relocation computations.
+//!
+//!  * A: The addend used to compute the value of the relocatable field.
+//!  * B: The base address at which a shared object is loaded into memory
+//!       during execution. Generally, a shared object file is built with a
+//!       base virtual address of 0. However, the execution address of the
+//!       shared object is different. See Program Header.
```

Comment:
Here and below: delete the 'See XXX' statements.

Reply:
Done